monitor/views.py

```diff
@@ -104,6 +105,21 @@ def queries_stats_ajax(request):
     return JsonResponse(json.load(urllib2.urlopen(req)))
 
 
+@login_required
+def api_usage_stats_ajax(request):
+    query = urllib.urlencode({
+        'query': 'api_client_username:%s' % (request.user.username),
+        'range': '604800',
+        'filter': 'streams:530f2ec5e4b0f124869546d0',
+        'interval': 'day'
+    })
+    req = urllib2.Request("http://mtg-logserver.s.upf.edu/graylog/api/search/universal/relative/histogram?%s" % (query))
+    base64string = base64.b64encode('%s:%s' % (settings.GRAYLOG_USERNAME,
+        settings.GRAYLOG_PASSWORD))
+    req.add_header("Authorization", "Basic %s" % base64string)
+    return JsonResponse(json.load(urllib2.urlopen(req)))
```

Comment:
also consider error cases here, where the query fails, and return an appropriate error message